docs/hacking/install.rst

```diff
@@ -131,15 +116,15 @@ system that Docker can be installed on:
 
    .. code-block:: bash
 
-      docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
-      docker run -d --name postgres -p 5432:5432 postgres
-      docker run -d --name redis -p 6379:6379 redis
-      docker run -d --name elasticsearch -p 9200:9200 -p 9300:9300 nickstenning/elasticsearch-icu
+      sudo docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
```

Comment:
I'm not sure about adding `sudo` here. It's entirely dependent on the configuration of docker whether or not using the docker CLI requires you to be root. Following the default instructions for setting up docker on OS X, for example, you don't need to be root to run `docker`.

This really boils down to so much of the struggle with writing documentation -- do we have a responsibility to document other people's systems (how to install and configure postgres, for example). In this case, I'd vote explain what you need to do to docker, and leave it up to people to work out how docker works on their system.